### DISCUSSION TITLE
Web Panel functionality and install script change requests

### DISCUSSION THREAD

**Discussion by Rebel-Sheep-Media-LLC** [jekhokie/raspberry-noaa-v2#397] [Ideas]
I would like to note a couple things and request a couple improvements. Firstly, I noticed that in the web panel, by default, 56 satellites are active. After taking the time to deselect all of these satellites and only enabling the 3 NOAA satellites I wanted to be displayed, (NOAA 15,18 and 19) I noticed that any time the panel was refreshed or I switched between the passes, captures and admin tabs, the panel would reset and default back to showing the original 56 satellites.. Meaning that I would have to go back and deselect all 56 satellites and re-enable my preferred satellites every time this happened, which is very frustrating. ALSO, another slightly frustrating issue that occurred during the install and configuration process,  I found it to be difficult to get the twitter push functionality working. The documentation covering twitter push was pretty vague, leading me to believe that once _twitter_push_ was set to "true", running _./install_and_upgrade.sh_ would automatically create the _~/.tweepy.conf_ file...That was not the case and after trial and error, I realized that _~/.tweepy.conf_ needed to be manually created. This action should be automatically done by the _install_and_upgrade.sh_ file. It would make the process much less confusing for some..

**Comment by Cadair** [comment]
Hey @Dzewa hope you don't mind but I converted this to a discussion.

The twitter suggestion is hopefully covered by #395 but if anyone wants to improve the docs in the mean time that would be great.

I am assuming when you are talking about the 56 satellites you are referring to the satellite tracker embedded at the top of the home page?

**Reply by Rebel-Sheep-Media-LLC** [reply]
That is correct. when I toggle the UI and select the satellite icon, it displays 3 menus. under "Enabled Satellites" it says "56".. and when I disable them and then only choose noaa 15,18 and 19.. it works until I refresh the page or go to a different tab (example: Admin tab).. the satellites refresh and show 56 satellites again, then I have to go back and remove all satellites and only select the ones I want to see. I realize that most probably don't pay much attention to the satellite and globe view, but I personally don't like the clutter of the unnecessary satellites I don't need to see.. and  It's just a pain in the butt to have to do this every single time

**Reply by Cadair** [reply]
I am not really familiar with that part of the UI as it's an external website we embed, I don't actually have it enabled.

I would assume there is no way for it to store state when it's iframed into our site, I will have to let others who might know more about it weigh in.

**Reply by Rebel-Sheep-Media-LLC** [reply]
Well, one other issue that comes to my mind since we are discussing the project anyway, is that line 8 inside the settings.yml file, i noticed that it shows "_time zone offset from UTC (for example, '-5' for US Eastern_" ... since I am located on the east coast (florida), I left that setting at the default of -5.. however, all of the passes I have seen so far are extremely noisy and there were some that the system couldnt even decode... I am 99.9% certain that my antenna is fine, since I was getting perfect results less than a week ago with an older setup.. All I have done was upgraded to V2... I have since changed that -5 valaue to -4, as -4 would be the correct value since daylight savings time is in effect.. Again, it wasn't an issue to fix once I figured out that it was the problem, but for newbies or first installs, it would be difficult to troubleshoot.. shouldn't the timezone offset be configured automatically be the software?

**Reply by Cadair** [reply]
I think there is an issue about that: #379 I haven't looked into it personally.

**Reply by dom-robinson** [reply]
Regarding the satvis issue its a limitation of pulling in the hosted feed from the satvis author. See my comments here: https://github.com/jekhokie/raspberry-noaa-v2/pull/130#issuecomment-804005656

Basically it could be hosted on the pi and the weather.tle limited to 'just' the wanted 4 satellites. But its a large block of code to add and the hosted version does work, but you do have to toggle out the unwanted satellites..

**Reply by Rebel-Sheep-Media-LLC** [reply]
@Cadair I have sent you an email, can you please take a sec to check that out.
